app/uk/gov/hmrc/emailverification/crypto/Decrypter.scala

```diff
@@ -16,25 +16,22 @@
 
 package uk.gov.hmrc.emailverification.crypto
 
+import javax.inject.{Inject, Singleton}
 import play.api.libs.json.{Json, Reads}
-import play.api.{Logger, LoggerLike}
-import uk.gov.hmrc.crypto.{Crypted, CryptoWithKeysFromConfig, Decrypter => HmrcDecrypter}
+import play.api.{Logger, LoggerLike, Play}
+import uk.gov.hmrc.crypto.{Crypted, CryptoWithKeysFromConfig}
 
 import scala.util.{Failure, Try}
 
-trait Decrypter {
-  def crypto: HmrcDecrypter
-
-  def logger: LoggerLike
-
+@Singleton
+class Decrypter @Inject()(){
+  private val config = Play.current.configuration.underlying
```

Comment:
Should inject the configuration

Reply:
have injected configuration into Decrypter